app/assets/stylesheets/application.css.scss

```diff
@@ -187,17 +148,47 @@ body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
 
 .main_panel {
   padding-left: 150px;
+  .thought-teasers{
+    margin: 10px 0;
+    .thought-headline{
+      background-repeat: no-repeat;
+      background-size: cover;
+      height: 150px;
+      width: 100vw;
+      position: relative;
+      left: -150px;
+      box-shadow: inset 0 -12px 100px 10px rgba(0, 0, 0, 0.4);
+
+      img {
+        height: 60px;
+      }
+
+      .title{
+        padding:200px;
+        text-align: right;
+        font-size: 2em;
+        color: #f9f9f9;
+      }
+
+      .body{
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 4<br>Opening curly brace `{` should be preceded by one space